docs/redis.md

```diff
@@ -6,11 +6,11 @@ If you would like to use [Cloud Memorystore](https://cloud.google.com/memorystor
 
 ```bash
 cd gke
-./create_redis.sh <REGION>
+./create_redis.sh <INSTANCE> <REGION>
```

Comment:
maybe use <ENV>  instead of <INSTANCE> as we use it in other places?

Update below that <ENV> is the name of an instance, which can be found as deploy/gke/<ENV>.yaml

Reply:
done